entries/m/mintos.com.json

```diff
@@ -8,6 +8,12 @@
     "documentation": "https://help.mintos.com/hc/en-us/sections/360000373858",
     "keywords": [
       "investing"
+    ],
+    "regions": [
+      "-fr",
+      "-gb",
+      "-in",
+      "-us"
```

Comment:
Why the `-` for the regions?

Reply:
Those regions will not appear in the country choices at sign-up.